File: src/main/java/io/github/thejaxonhill/tarotapi/TarotCard.java
package io.github.thejaxonhill.tarotapi;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TarotCard {

    private Integer id;

    private String type;

    private String shortName;

    private String name;

    private String value;

    private int intValue;

    private String suit;

    private String upMeaning;

    private String revMeaning;

    private String desc;

    private String imageLink;

}

File: src/main/kotlin/io/github/thejaxonhill/tarotapi/domain/TarotCard.kt
package io.github.thejaxonhill.tarotapi.domain

class TarotCard (
    val id: Int,
    val type: String,
    val shortName: String,
    val name: String,
    val value: String,
    val intValue: Int,
    val suit: String? = null,
    val upMeaning: String,
    val revMeaning: String,
    val desc: String,
)